doc/whats_new/v0.20.rst

```diff
@@ -22,6 +22,11 @@ random sampling procedures.
 - :class:`neural_network.MLPRegressor` (bug fix)
 - :class:`neural_network.MLPClassifier` (bug fix)
 
+- Classes derived from :class:`ensemble.BaseBagging`. The attribute
```

Comment:
No, wrong section. The model is identical. The API has changed. Belongs under API changes summary below